hackathon_site/event/jinja2/event/base.html

```diff
@@ -11,6 +11,9 @@
         <!-- Compiled and minified CSS for Materialize -->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" />
 
+        <!-- Google Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,500,700,900|Nunito:400,500,700,900&display=swap" rel="stylesheet">
+
         <title>{% block title %}Hackathon Site{% endblock %}</title>
```

Comment:
Per https://github.com/ieeeuoft/makeuoft-site-2021/pull/8#discussion_r549034564, let's update this to be built from the `HACKATHON_NAME` and `EVENT_START_DATE` settings